src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -165,4 +163,68 @@
         DISTINCT(attribute_name)
         FROM ${tableName};
     </select>
+
+    <!-- Grab Total Profiled Counts -->
+    <select id="getTotalProfiledCounts">
+        SELECT
+        gene as hugoGeneSymbol,
+        COUNT(*) as numberOfProfiledCases
+        FROM sample_to_gene_panel stgp
+            INNER JOIN gene_panel_to_gene gptg on stgp.gene_panel_id = gptg.gene_panel_id 
+        <where>
+            stgp.alteration_type = '${alterationType}' 
+            AND
+            sample_unique_id IN ( <include refid="sampleUniqueIdsFromStudyViewFilter"/>
+            <if test="applyPatientIdFilters == true">
+                INTERSECT <include refid="getSampleIdsFromPatientIds"/>
+            </if>
+            )
+        </where>
+        GROUP BY gptg.gene;
+    </select>
+
+    <!-- Helper query to get counts for totalProfiledCount when no genePanel data is available-->
```

Comment:
add in comment the detail that when no genePanel we consider it whole genome sequencing?